cmd/kops/create_cluster.go

```diff
@@ -1054,6 +1056,11 @@ func completeOpenstackLBSubnet(cmd *cobra.Command, args []string, toComplete str
 	return nil, cobra.ShellCompDirectiveNoFileComp
 }
 
+func completeOpenstackOctaviaProvider(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	// TODO call into cloud provider to get list of Octavia providers
+	return nil, cobra.ShellCompDirectiveNoFileComp
```

Comment:
Is there a static list of common Octavia providers we can offer for completion? It looks like `octavia` is a possible value.

Reply:
Added completion based on the list provided in this document.
https://docs.openstack.org/octavia/latest/admin/providers/index.html

`octaiva` is not listed in the document but is the default provider for older version of Octavia, so I've also included it in the list.